docs/mods/armaforces_medical.md

```diff
@@ -50,44 +50,4 @@ Osoby przytomne mogą używać nowego środka przeciwbólowego - Apapu. Został
 | Siła przeciwbólowa        | 0.6                    | 0.8                           |
 | Zastosowanie              | Ból u przytomnych osób | Duży ból lub u nieprzytomnych |
 | Efekty uboczne            | Brak                   | Obniżone tętno                |
-| Ryzyko nieprzytomności    | Pomijalne              | Wysokie, wystarczą 2 użycia   |
-
-## Opaski uciskowe
-
-Opaski uciskowe (znane też jako stazy, tourniquets) pozwalają na zatamowanie krwawienia z kończyn pacjenta.
-
-- zakładane tylko na ręce i nogi,
-- drastycznie szybsze w użyciu od bandażowania (kupują czas medykowi na opatrzenie torsu i głowy, piechocie pozwalają na nieprzerwane prowadzenie ognia w przypadku obrażeń kończyn),
-- wielokrotnego użytku - po ściągnięciu wracają do ekwipunku ściągającego,
-- po kilku minutach od założenia powodują narastający ból,
-- wlewy medyczne (krew, morfina) nie przynoszą efektu przy podaniu drogą zastazowanej kończyny.
-
-## Procedura leczenia
-
-### 1. Zadbaj o swoje bezpieczeństwo
-
-Nigdy nie rozpoczynaj leczenia na środku drogi pod ostrzałem. Postaraj się zadbać o bezpieczną pozycję do rozpoczęcia działań. W razie potrzeby można przeciągnąć lub przenieść rannego w bezpieczniejsze miejsce. Jeżeli chcemy go przenosić (co trwa dłużej niż przeciąganie) powinniśmy najpierw spróbować zastazować mu rany.
-
-### 2. Nieprzytomny dowódca
-
-Jeśli dowódca jest nieprzytomny lub martwy podnieś jego radio. Do czasu odzyskania przytomności przez dowódcę przejmij dowodzenie lub wyznacz inną osobę do tej roli.
-
-### 3. Leczenie ran
-
-Jeśli pacjent ma rozległe rany wielu kończyn warto zastosować stazy (tourniquets) dopóki rany nie zostaną zabandażowane. Następnie zabandażuj wszystkie rany. Pacjent musi przestać krwawić.
-
-### 4. Reanimacja
-
-Sprawdź czy pacjent ma puls.
-
-#### Brak pulsu
-
-Oceń ilość krwi. Jeśli pacjent stracił dużo krwi warto rozpocząć przetaczanie krwi.
-Rozpocznij wykonywanie RKO (CPR). Po każdym wykonaniu sprawdź czy nie wrócił puls. Jeśli pacjentem zajmuje się więcej osób to podczas sprawdzania pulsuupewnij się, że nikt nie wykonuje RKO, ponieważ możesz odczytać fałszywy puls. Jeśli po kilku próbach puls nadal nie wraca przetocz więcej krwi.
-Jeśli nie jesteś medykiem wykonuj RKO (CPR), aż do przybycia pomocy lub odzyskania pulsu. Każdorazowa akcja reanimacyjna wydłuża czas do śmierci.
-
-#### Jest puls
-
-Jeśli puls jest słaby (<45BPM) zastosuj adrenalinę (Epinephrine).
-Jeśli pacjent stracił dużo krwi i się nie budzi - przetocz krew.
-Zaszyj rany. Jeśli nie jesteś medykiem lub nie masz zestawu do szycia regularnie sprawdzaj czy żaden z bandaży się nie otworzył. Pacjent może się wykrwawić i nikt może tego nie zauważyć.
+| Ryzyko nieprzytomności    | Pomijalne              | Wysokie, wystarczą 2 użycia   |
```

Comment:
Always end file with one new line. This prevents cases like here, you did not change anything in this line but it is marked as changed.
```suggestion
| Ryzyko nieprzytomności    | Pomijalne              | Wysokie, wystarczą 2 użycia   |

```